<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roposset</class>
 <widget class="QWidget" name="roposset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前选择：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_current">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_other">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>620</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>110</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>110</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>110</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>110</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>620</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>13</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>13</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
